_posts/2023-09-11-notes-from-community-call-7-september-2023.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Notes from the Standard for Public Code community call - 7 September 2023"
+date: 2023-09-11
+author: Jan Ainali
+type: blogpost
+excerpt: Reviews in two days needs clarification on how it will be tested
+categories:
+  - Community call
+---
+
+# 7 September 2022 Standard for Public Code community call
+
+## Attendees
+
+* [Jan Ainali](https://publiccode.net/who-we-are/team/jan-ainali.html)
+* [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html)
+* [Rasmus Frey](https://www.os2.eu/om-os2)
+
+## Updates from the Foundation
+
+* Two new codebases published their assessments towards the Standard for Public Code:
+  * [The open source repository template from the Swedish Agency for Digital Government]( https://github.com/diggsweden/open-source-project-template/blob/main/docs/standard-for-public-code-assessment.html)
+  * [Circulaw - a knowledge platform that encourages positive circular economy regulation](https://github.com/Dark-Matter-Labs/circulaw/blob/main/docs/circulaw-standard-for-public-code.html)
+
+## Background
+
+In [Require review of contributions](https://standard.publiccode.net/criteria/require-review-of-contributions.html) we have the requirement:
+
+> Reviews SHOULD happen within two business days.
+
+But this is ambiguous in terms of what it means.
+Should it be finished withing two days or just started?
+Some reviews will reasonably take longer time because of the size and complexity of the contribution.
+How can we make this less ambiguous, but also not encourage started reviews that are not meaningful (compare to the [British "Hello nurses"](https://www.bmj.com/content/315/7101/143.6))?
+See [issue #908](https://github.com/publiccodenet/standard/issues/980).
+
+## Notes
+
+We first noted that having a service level agreement that has a development vendor on standby to review contributions in two business days might be costly.
+As some codebases are in heavier development than others and might have developers at hand, could the requirement be differentiated somehow?
+While the notion about cost is true, with the recently added requirement in [Welcome contributors](https://standard.publiccode.net/criteria/welcome-contributors.html):
+
+> The contribution guidelines SHOULD document who is expected to cover the costs of reviewing contributions.
+
+it should be clear that it may not be the one reviewing that should pay for it.
+This means that the Standard allows for different situations already and if all review is outsourced, the cost for that *could* be on the party making a contribution.
```

Comment:
This seems like it would be absolutely chilling to volunteer or amateur contributors.

Reply:
If the policy says that every contribution has to be like that, sure. But it could, of course, also be more nuanced and differentiated. Either way, this is already in the standard.